website/static/css/pages/search-page.css

```diff
@@ -106,3 +106,7 @@ div.search-result  {
 .search-contributor-links {
     padding-top: 10px;
 }
+
+.osf-search {
+    z-index: 1029 !important;
```

Comment:
Using the `!important` directive is generally a bad idea. Instead, consider increasing the specificity of the rule's selector.